x/exchange/keeper/commitments.go

```diff
@@ -426,3 +457,43 @@ func (k Keeper) consumeCommitmentSettlementFee(ctx sdk.Context, req *exchange.Ms
 	antewrapper.ConsumeMsgFee(ctx, exchangeFees.ExchangeFees, req, "")
 	return nil
 }
+
+// validateTransferCommitments which validate TransferCommitments message request.
+func (k Keeper) validateTransferCommitments(
+	ctx sdk.Context,
+	store storetypes.KVStore,
+	req *exchange.MsgMarketTransferCommitmentsRequest,
+	account sdk.AccAddress,
+) error {
+	// Blocked address check
+	if k.bankKeeper.BlockedAddr(account) {
+		return fmt.Errorf("%s is not allowed to receive funds", account)
+	}
```

Comment:
We don't need to check this; no funds are actually being moved, so we don't need to worry about a blocked account receiving any funds.